src/renderer/components/uielements/assets/assetIcon/AssetIcon.tsx

```diff
@@ -19,6 +31,9 @@ type ComponentProps = {
 type Props = ComponentProps & React.HTMLAttributes<HTMLDivElement>
 
 export const AssetIcon: React.FC<Props> = ({ asset, size = 'normal', className = '', ...rest }): JSX.Element => {
+  const { network$ } = useAppContext()
+  const network = useObservableState<Network>(network$, DEFAULT_NETWORK)
```

Comment:
Pls provide network as parameter. We have a rule (you might not know it): Access any `XYZContext` in `XYZViews` only (but not in components).